assets/js/stripe-payment-request.js

```diff
@@ -548,6 +548,12 @@ jQuery( function( $ ) {
 		},
 
 		attachPaymentRequestButtonEventListeners: function( prButton, paymentRequest ) {
+			// First, attach a flag setting event so we can know when a PRB has initiated payment
+			prButton.on( 'click', function ( evt ) {
+				$( 'body' ).addClass( "woocommerce-stripe-prb-clicked" );
+			});
```

Comment:
Similarly to my other comment, we should probably include _why_ we need to know when a PRB initiated a payment 🙂